extensions/Cake.Buns.ReportPortal.yml

```diff
@@ -3,8 +3,8 @@ Name: Cake.Buns.ReportPortal
 NuGet: Cake.Buns.ReportPortal
 Assemblies:
 - /**/Cake.Buns.ReportPortal.dll
-Repository: https://github.com/OlegYanushkevich/Cake.Buns.ReportPortal.git
-ProjectUrl: https://github.com/OlegYanushkevich/Cake.Buns.ReportPortal/
+Repository: https://github.com/OlegYanushkevich/Cake.Buns.ReportPortal/
```

Comment:
Repository / GitHub user seems to be removed.

@Jericho Isn't there a check in addin discoverer to avoid adding links leading to a 404 (or was the user removed just after addin discoverer was run)?

Reply:
We do validate the ProjectUrl and replace it with a link to the nuget package when we detect HTTP 404. That's why the `ProjectUrl` in this YAML was changed:
- from `https://github.com/OlegYanushkevich/Cake.Buns.ReportPortal/`
- to `https://www.nuget.org/packages/Cake.Buns.ReportPortal/`. 
 
However, the same logic is not applied to the repository URL. That's why the AddinDiscoverer did not update the `RepositoryUrl` (other than `.git` which the author seems to have removed in their nuspec).